cves/CVE-2015-6780.yml

```
@@ -4,14 +4,14 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: Other
```

What does other mean? Is there not a CWE associated with this?

---

Under the "Technical Details" Section where my other CVE had it's CWE listed "Other (NVD-CWE-Other)" was the only thing there. However on double checking I noticed this CVE has an additional section that lists this as "CWE-416: Use After Free". I must have missed that the first time.
```suggestion
CWE: 416
```